canonical-web-and-design/ubuntu.com
# fix typo

## Done

"conterisation" => "containerisation" (pretty sure the former isn't a word)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/networking
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that "containerisation" is spelled correctly
